content/docs/studio/user-guide/views/view-settings.md

```diff
@@ -104,15 +104,26 @@ they are not saved into Git. It does not access any other data in your remote
 storage. And you do not need to provide the credentials if any DVC data remote
 in not used in your Git repository.
 
-### Tracking scope
+### Mandatory columns
 
-DVC Studio can track upto 200 metrics, parameters, and files. If you have more
-than 200 values in your Git repository, you can specify which ones to track and
-which ones to leave out. To ensure that a value is included, make sure that it
-is selected in the tracking scope. Any value that is not selected may not
-display in your view.
+If your repository has more than 200 columns, you can specify which columns are
```

Comment:
Small detail: This first sentence can be interpreted as "Only after you exceed 200 columns, the settings for mandatory columns become available".

Maybe we can use the same explanation as in https://github.com/iterative/viewer/pull/2621/ for consistency?

Reply:
Have changed.